nlp_primitives/utilities.py

```diff
@@ -1,19 +1,23 @@
 import string
+from typing import List
 
 import nltk
 
 
-def clean_tokens(textstr):
-    wn = nltk.WordNetLemmatizer()
-
-    textstr = textstr.translate(str.maketrans("", "", string.punctuation))
-    textstr = [ch for ch in textstr.split(" ") if len(ch) > 0]
+def clean_tokens(text: str) -> List[str]:
```

Comment:
took this opportunity to refactor this function. happy to revert changes if anything is worse now!